### PR TITLE
changes pertaining to the payment domain

### PR DESCRIPTION
**Pull request by lturihabwe**

### CODE REVIEW

File: backend/src/main/java/miu/edu/waa/backend/domain/Payment.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    @Id
+    @GeneratedValue
+    private Long id;
+    private Long cardNumber;
+    private LocalDate expiryDate;
+    private LocalDate createdAt;
+    private Integer cvv;
+    @ManyToOne
+   // @JoinColumn(name = "buyer_id")
```
Comment (joshNic) [inline backend/src/main/java/miu/edu/waa/backend/domain/Payment.java:23]: @lturihabwe remove comment

**Review by joshNic — CHANGES_REQUESTED** [review]
@lturihabwe handle feedback

**Review by joshNic — CHANGES_REQUESTED** [review]
@lturihabwe handle feedback

**Review by FahdJamy — APPROVED** [review]

**Review by joshNic — APPROVED** [review]
LGTM

**Pull request merged**